awx/ui_next/src/screens/Job/JobDetail/JobDetail.jsx

```diff
@@ -274,45 +264,15 @@ function JobDetail({ job, i18n, history }) {
             </LaunchButton>
           )}
         {job.summary_fields.user_capabilities.delete && (
-          <Button
-            variant="danger"
-            aria-label={i18n._(t`Delete`)}
-            onClick={() => setIsDeleteModalOpen(true)}
+          <DeleteButton
+            name={job.name}
+            modalTitle={i18n._(t`Delete Job`)}
+            onConfirm={deleteJob}
           >
             {i18n._(t`Delete`)}
-          </Button>
+          </DeleteButton>
         )}
-      </ActionButtonWrapper>
-      {isDeleteModalOpen && (
```

Comment:
awesome that we can delete this and take advantage of code built in to the DeleteButton component!